test/test_open.py

```diff
@@ -183,6 +203,11 @@ def test_mode_t(self, safer_open):
             fp.write('hello')
         assert FILENAME.read_text() == 'hello'
 
+    # Skip if we are on Windows and do not have admin rights
+    @unittest.skipIf(
```

Comment:
Quibble: extract the common variable!

```
IS_WINDOWS_USER = os.name == 'nt' and not is_windows_admin()
skip_windows = unittest.skipIf(
    IS_WINDOWS_USER, 'This test requires admin privileges to create symlink files on Windows'
)

[....]

    @skip_windows
     def test_something():
```

Reply:
I definitely learned something new here!